client/rpc/src/lib.rs

```diff
@@ -272,6 +297,75 @@ where
         Ok(read_fs.root())
     }
 
+    async fn is_file_in_forest(&self, key: Option<String>, file_key: H256) -> RpcResult<bool> {
```

Comment:
Should we call the `key` parameter `forest_key`? So that it is clearer. `key` alone feels a bit confusing on what key is it. Given that we have so many keys.

Reply:
I wanted to keep the naming convention of the other RPC methods. I agree with you though, it was a bit confusing for me just calling it `key`. Should I change all of them?

Reply:
Went ahead and changed them all 😬